workshops/test/test_tasks.py

```diff
@@ -70,6 +71,12 @@ def test_task_detail_view_reachable_from_event_person_and_role_of_task(self):
         response = self.client.get(reverse('task_details', args=[str(correct_task.id)]))
         assert response.context['task'].pk == correct_task.pk
 
+    def test_task_wrong_url_not_accepted(self):
+        task = self.fixtures['test_task_1']
+        task.url = 'htp://www.example.com'
+        task_form = TaskForm(initial={'task':task})
+        self.assertFalse(task_form.is_valid())
```

Comment:
@narayanaditya95 thanks!

1. It's a good habit to write docstrings to your tests because they're showed by the TestRunner when tests fail.

2. I'll merge this when we open v1.7 for merging.